src/NzbDrone.Core/MediaCover/ImageResizer.cs

```diff
@@ -15,25 +18,20 @@ public class ImageResizer : IImageResizer
         public ImageResizer(IDiskProvider diskProvider)
         {
             _diskProvider = diskProvider;
+
+            // More conservative memory allocation
+            SixLabors.ImageSharp.Configuration.Default.MemoryAllocator = new SimpleGcMemoryAllocator();
         }
```

Comment:
```
            // Thumbnails don't need super high quality
            SixLabors.ImageSharp.Configuration.Default.ImageFormatsManager.SetEncoder(JpegFormat.Instance, new JpegEncoder()
            {
                Quality = 92
            });
```

Series: The 100
- poster.jpg 728k (source image)
- poster-500.jpg 43.9k original sonarr (q90 i think) vs 55.8k imagesharp (q92)
- poster-250.jpg 12.7k original sonarr (q90 i think) vs 16.2k imagesharp (q92)

q90 produces like 44.5k for poster-500.jpg so slightly larger than gdiplus, at comparable quality (visually inspected).
q92 reduces artifacts by 30% or so, not sure if it's worth the 20% bump in size.

I still dunno if we should go for q90 or q92, but it's better than the 120k file at default q.
If you really want good quality then you probably have to go for q95, but remember that retina screens are going to use the poster.jpg anyway since the browser selects the right image based in ppi.